@ecomplus/storefront-template/template/js/netlify-cms/preview/settings/general.js

```diff
@@ -0,0 +1,145 @@
+import BasePreview from '../base-preview'
+import virtualDoc from '../virtual-doc'
+import fetchPage from '../fetch-page'
+
+const themesUrl = 'https://cdn.jsdelivr.net/gh/ecomplus/storefront@themes-dist'
+
+const fetchCssTheme = (theme) => {
+  if (theme === '_') {
+    return Promise.resolve('')
+  }
+  return window.fetch(`${themesUrl}/${theme}.css`)
+}
+
+export default class CodePreview extends BasePreview {
+  constructor() {
+    super()
+    this._ = {
+      ...this._,
+      state: {
+        title: 'Geral',
+      },
+      route: {
+        path: '/'
+      }
+    }
+
+    this.customEntrys = [
+      {
+        'type': 'banner-slider',
+        'autoplay': 9000,
+        'slides': [
+          {
+            'img': 'https://storefront-template.netlify.app/img/uploads/banner2.webp',
+            'link': '/headset',
+            'alt': 'Headset banner',
+            'mobile_img': 'https://storefront-template.netlify.app/img/uploads/banner2.webp'
+          }
+        ]
+      },
+      { 'type': 'info-bar', 'enabled': true },
+      {
+        'type': 'collection-shelf',
+        'shuffle': false,
+        'collection_id': null,
+        'sort': 'offers',
+        'headless': false
+      }
+    ]
```

Comment:
Isso não é customizado em `settings/general`, ainda não entendi porque isso deveria ser renderizado com EJS aqui (como se fosse variável)...

Reply:
Além de não precisar, isso vai fazer ficar diferente do que está salvo na homepage também

Reply:
é customização da pagina inicial, sem essas entrys a pagina fica quebrada. e como to usando a pagina inicial pra exibir como preview no settings isso é necessário.

Reply:
Se você parsear o EJS essas entries são obrigatórias mesmo (claro), o ponto é que não é necessário (nem bom) parsear o EJS da página inicial aqui, basta simplesmente usar o HTML já parseado...